### PR TITLE
[ServiceBus] Files for SB Perf Tests

### PR DESCRIPTION
**Pull request by kashifkhan**
* I've set preloads to 20,000 instead of a million due to the time it takes to populate
* Message size is left the same

### CODE REVIEW

**Review by swathipil — APPROVED** [review]
lgtm!

File: sdk/servicebus/perf-tests.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Service: servicebus
+
+Project: sdk/servicebus/azure-servicebus
+
+PrimaryPackage: azure-eventhub
+
+PackageVersions:
+- azure-core: 1.26.1
```
Comment (pvaneck) [inline sdk/servicebus/perf-tests.yml:8]: Perhaps consider using the latest azure-core (`1.26.3`).

File: sdk/servicebus/perf-tests.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  azure-eventhub: 7.8.3
```
Comment (pvaneck) [inline sdk/servicebus/perf-tests.yml:9]: This should be `azure-servicebus: 7.8.3` right?

File: sdk/servicebus/perf-tests.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+- azure-core: source
+  azure-eventhub: source
+
+Tests:
+- Test: send-queue-message-batch
+  Class: SendQueueMessageBatchTest
+  Arguments:
+  - --event-size 1024 --batch-size 100 --parallel 64
```
Comment (pvaneck) [inline sdk/servicebus/perf-tests.yml:17]: Is `event-size` the correct argument name or a carryover from the eventhubs perf configuration?

File: sdk/servicebus/perf-tests.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  - --event-size 1024 --batch-size 100 --parallel 64 --uamqp-transport
+  - --event-size 1024 --batch-size 100 --parallel 64 --transport-type 1
+  - --event-size 1024 --batch-size 100 --parallel 64 --transport-type 1 --uamqp-transport
+
+- Test: receive-queue-message-batch
+  Class: ReceiveQueueMessageBatchTest
+  Arguments:
```
Comment (pvaneck) [inline sdk/servicebus/perf-tests.yml:24]: Optional, but you can use yaml anchors/aliases for duplicate argument lists. Example usage: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/storage/azure-storage-blob/perf-tests.yml#L16

**Review by pvaneck — COMMENTED** [review]
A few comments inline. Is this PR superseding https://github.com/Azure/azure-sdk-for-python/pull/28926?

File: sdk/servicebus/perf-tests.yml (hunk 3 above)
Comment (kashifkhan) [reply sdk/servicebus/perf-tests.yml:17]: No, Ive copied over the info from Anna's PR to fix it :)

File: sdk/servicebus/perf-tests.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  azure-servicebus: source
+
+Tests:
+- Test: send-queue-message-batch
+  Class: SendQueueMessageBatchTest
+  Arguments:
+  - --message-size 10240 --batch-size 10 
+  - --message-size 10240 --batch-size 10  --uamqp-transport
+  - --message-size 10240 --batch-size 10  --transport-type 1
+  - --message-size 10240 --batch-size 10  --transport-type 1 --uamqp-transport
```
[the remark this answers is not in the document]
Comment (kashifkhan) [reply sdk/servicebus/perf-tests.yml:20]: good catch, these need to go as well

File: sdk/servicebus/perf-tests.yml [hunk 6]
```diff
@@ -0,0 +1,44 @@
+Service: servicebus
+
+Project: sdk/servicebus/azure-servicebus
+
+PrimaryPackage: azure-eventhub
```
Comment (pvaneck) [inline sdk/servicebus/perf-tests.yml:5]:
```suggestion
PrimaryPackage: azure-servicebus
```

**Review by pvaneck — APPROVED** [review]
Just one correction inline. otherwise looks good!

**Pull request merged**